File: backend/backend.Domain/Model/Column.cs
using backend.Domain.Model.Abstract;

namespace backend.Domain.Model
{
    public class Column : Entity
    {
        public string Name { get; set; }
        public int MaxTasks { get; set; } = 15;
        public int Order { get; set; } = 0;
        public Guid BoardId { get; set; }

        public Board Board { get; set; }
        public ICollection<Task> Tasks { get; set; }
    }
}

File: frontend/src/app/model/column.ts
export class Column {
    public id? : string;
    public name? : string;
    public maxTasks? : number;
    public order? : number;

    constructor(id : string, name : string, maxTasks : number, order : number) {
        this.id = id;
        this.name = name;
        this.maxTasks = maxTasks;
        this.order = order;
    }
}